pkg/notifications/notificationscenter/notification_center.go

```diff
@@ -0,0 +1,164 @@
+package notificationscenter
```

Comment:
I think `pkg/notifications/center/notification_center.go` and `packer center` should be enough. Or do you think it can cause confusion?